src/threads/thread.c

```diff
@@ -72,8 +71,6 @@ static void schedule (void);
 void thread_schedule_tail (struct thread *prev);
 static tid_t allocate_tid (void);
 
-int load_avg;
-
```

Comment:
Odd. I thought we would still try to use these variables. They're removing the implementation everywhere